app/src/main/res/values/preference_keys.xml

```diff
@@ -27,6 +27,7 @@
     <string name="pref_key_performance_block_images" translatable="false"><xliff:g id="preference_key">pref_performance_block_images</xliff:g></string>
 
     <string name="pref_key_default_browser" translatable="false"><xliff:g id="preference_key">pref_default_browser</xliff:g></string>
+    <string name="pref_key_leakcanary" translatable="false">pref_key_leakcanary</string>
```

Comment:
nit: Not really needed since these strings are not intended to be translated but in the rest of the file xliff is used so for consistency it should be used also.

Reply:
Done